org.eclipse.jdt.core.manipulation/proposals/org/eclipse/jdt/internal/ui/text/correction/AddAllMissingJavadocTagsProposalCore.java

```diff
@@ -0,0 +1,180 @@
+package org.eclipse.jdt.internal.ui.text.correction;
```

Comment:
Missing license.  A few others are missing licenses as well.  Other than that, looks good.